Examples/JamitFoundationExample/App/Sources/ActionView/ActionViewControllerViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
+import JamitFoundation
+
+struct ActionViewControllerViewModel: ViewModelProtocol {
+    let imageURL: URL?
+
+    init(imageURL: URL? = ActionViewControllerViewModel.default.imageURL) {
+        self.imageURL = imageURL
+    }
+}
+
+extension ActionViewControllerViewModel {
+    static let `default`: ActionViewControllerViewModel = .init(
```

Comment:
I think `: Self = .init(...)` is more compact.

Reply:
Done.